functions/migration_opts_default.pp

```diff
@@ -0,0 +1,12 @@
+function peadm::migration_opts_default () {
```

Comment:
none of the new functions are documented :(

Reply:
I think these are relatively simple functions for just holding an object, and didn't feel they needed documented.